torch/distributed/rpc/api.py

```diff
@@ -58,6 +64,42 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# States used by `def _wait_all_workers()`.
+# `_ALL_WORKER_NAMES` is initialized on initiaizing RPC layer.
+_ALL_WORKER_NAMES = None
+# `_SHUTDOWN_INTENT_WORKER_NAMES` is an empty set at beginning.
+# It's only used by leader worker. Leader worker is elected as the first
+# worker in a sorted worker name list.
+# Whenever there is a worker showing shutdown intention to the leader,
+# leader add this worker's name to the set.
+# This set includes the leader's name, for marking that `_wait_all_workers()`
```

Comment:
This isn't completely clear to me, since this says the set includes the leader's name, but it's initially empty. Could we clarify to say that the leader's name itself is also added to the set, when the leader calls `wait_all_workers`?